src/Controller/ArticlesController.php

```diff
@@ -80,6 +80,12 @@ public function textAction(Request $request, string $id, int $version = null) :
                 return true;
             });
 
+        // When NUMBER of comments > 0, text must be:
+        // '<span aria-hidden="true">NUMBER</span><span class="visuallyhidden">Open annotations (there are currently NUMBER annotations on this page).</span>'
+        // When 0 comments, text must be:
+        // "<span aria-hidden=\"true\">&#8220;</span><span class=\"visuallyhidden\">Open annotations (there are currently 0 annotations on this page).</span>
+        $arguments['hypothesisOpenerAffordance'] = ViewModel\Button::speechBubble(
+          '<span aria-hidden="true">12</span><span class="visuallyhidden">Open annotations (there are currently 12 annotations on this page).</span>', true, null, null, true, 'HypothesisOpenerAffordance');
```

Comment:
Not a fan of knowing about the JS at this point (it's like knowing class names). Should abstract it in patterns-php.